src/main/java/org/jitsi/jigasi/CallManager.java

```diff
@@ -25,21 +25,83 @@
 import java.util.concurrent.*;
 
 /**
- * FIXME: copied from Jitsi
+ * CallManager responsible for processing call operations like answer or hangup
+ * in new thread using a manged pool of threads.
+ *
+ * @author Pawel Domas
+ * @author Damian Minkov
  */
 public class CallManager
 {
-
     private final static Logger logger = Logger.getLogger(CallManager.class);
 
     private static final int POOL_SIZE = 5;
 
+    /**
+     * The thread pool to serve all call operations like answer and hangup.
+     * We initialize it with the 5 threads and can grow up to 4 time that size.
+     */
     private static ExecutorService threadPool
-        = Executors.newFixedThreadPool(POOL_SIZE);
+        = new ThreadPoolExecutor(
+                POOL_SIZE, POOL_SIZE * 4,
```

Comment:
Why do we need minimum 5 threads? They don't do a lot of work, and 1 should suffice.

Reply:
Ok, I will update it.